structure-message/requirements/browse-by-topic.md

```diff
@@ -0,0 +1,83 @@
+
+### Query 1: Get the list of statistical domains and the dataflows attached to them
+
+Required artefacts: Category Scheme, Dataflow
+
+The starting point is to retrieve the list of *topics* for which data are available. These are known as `category schemes` in SDMX. The `dataflows` attached to each category give information about the data that can be retrieved for each of the *topics*. Both the category schemes and the dataflows can be retrieved using a categoryscheme query, along with the `references` parameter:
+
+```
+https://ws-entry-point/categoryscheme?references=parentsandsiblings
+```
+
+Alternatively, if only the topics of a particular category scheme are needed, it is possible to only retrieve that particular category scheme (assuming we also know the maintenance agency for the category scheme):
+
+```
+https://ws-entry-point/categoryscheme/agency-id/category-scheme-id?references=parentsandsiblings
+```
+
+The response will contain the `category scheme(s)` and its `categories`, as well as the dataflows attached to them.
+
+The screenshot below shows an example of the type of user interface (a treeview control in this case) that can be built from a category scheme, using the [ECB Statistical Data Warehouse](https://sdw.ecb.europa.eu) as an example.
+
+![Treeview control](https://github.com/sdmx-twg/sdmx-json/raw/master/structure-message/requirements/img/cs-treeview.png)
+
+The screenshot below displays the category scheme as a list box, similar to the typical control displayed by mobile apps. The example is taken from the ECB statistical tablet app.
+
+![List of statistical domains](https://raw.githubusercontent.com/sdmx-twg/sdmx-json/master/structure-message/requirements/img/cs-list.png)
+
+The client typically needs to display the names of the categories. In addition, some clients might also want to display the descriptions of the categories and the name of the category schemes and possibly the name of its maintenance agency. Some clients might also want to display the number of dataflows attached to each category.
+
+Once a user has selected a category, the client could display the list of dataflows attach to the category. The screenshot below shows an example of the type of user interface (a list box in this case) that can be built from the response. The example is taken from the ECB statistical tablet app.
+
+![List of dataflows](https://raw.githubusercontent.com/sdmx-twg/sdmx-json/master/structure-message/requirements/img/df-list.png)
+
+The client typically needs to display the names of the dataflows. In addition, some clients might also want to display the descriptions of the dataflows and the name of the selected category.
+
+In order to perform the next query, clients will need the full references to the dataflows (id, agency id and version).
+
+### Query 2: Find data in the selected dataflow, using concept filters
+
+Required artefacts: Content Constraint
```

Comment:
My understanding is Data Availability is the web service to answer the question about what data exists for a Dataflow. 
https://github.com/sdmx-twg/sdmx-rest/wiki/Data-Availability

Reply:
@agent96 Thanks for highlighting. Will be resolved with ticket https://github.com/sdmx-twg/sdmx-json/issues/86.